### DISCUSSION TITLE
AtlasOS banned in Minecraft Ranked Bedwars

### DISCUSSION THREAD

**Discussion by oWqnd** [Atlas-OS/Atlas#1192] [Suggestions]
![Screenshot_20240819-122845__01](https://github.com/user-attachments/assets/3e049313-afbf-40fb-ac71-a83b7898138e)

As you can see in the screenshot that from now on AtlasOS is banned in Ranked Bedwars. Because, if someone thinks you're cheating and SS (Screenshare) you, then a Ranked Bedwars SSer will SS you and some services are disabled in AtlasOS that makes it impossible to Screenshare (as the screenshot says). So I would suggest turning those services on by defualt.

**Comment by LumiFae** [comment]
Those services are on by default, you can legit screenshare on atlas?????

**Reply by oWqnd** [reply]
idk then why did he post that thing?

**Reply by LumiFae** [reply]
i dont know, tell them that it is possible to screenshare on atlas. they should make their rules more specific and say if you cant screenshare then you aren't allowed to use that custom os or whatever

**Comment by oWqnd** [comment]
aight

**Comment by he3als** [comment]
Atlas does turn off certain services by default, from which screensharers use the data to see which processes have been run. However, they are not permanently removed and can be re-enabled manually. v0.5 will have a script to re-enable these services.

**Reply by oWqnd** [reply]
👍

**Comment by Ast3risk-ops** [comment]
That is really dumb, the admins are brain-dead idiots. This shows their complete lack of competence.

Linux doesn't automatically equal cheating. Garbage tools like BAC (Badlion Anti-Cheat) or Echo.ac (invasive spyware) aren't available for Linux because the developers don't care.

They need to adopt the Hypixel model for this problem, hackers exist on windows and Mac too. I would honestly stop playing on that server entirely because with this you can see the admins have no fucking clue what they're doing.

The Hypixel model is to just deal with hackers as they appear with server side anticheat and community moderation (Hypixel Atlas), no scanning computers or moderating operating systems.

**Reply by oWqnd** [reply]
true

**Reply by Ast3risk-ops** [reply]
Just tell the admins they're wrong and need to change their strategy to something that works better (Hypixel model) and if they don't listen ditch the server. That's my two cents.

**Comment by oWqnd** [comment]
I made a ticket and told that atlas is adding a script in v0.5 to enable necessary services that are required for ssing. Let's see what they do.

**Reply by Ast3risk-ops** [reply]
They shouldn't have to ask for a screen share to prove this garbage. A server side anticheat is good enough.

**Comment by scanash00** [comment]
Tomorrow is older than that Moon person, you can screenshare on Atlas, if that person was smart enough they would now, and Atlas is also open source, they would know what it does

**Reply by Ast3risk-ops** [reply]
Every bedwars competetive server does this. Digging around on the Hypixel forums tells me it involves taking control of your PC. The common argument is "don't play it if you hate ssing" which is overused and weak. I think people should spend their brain cells looking for better solutions instead of spouting the same shit.

My better solution is a serverside anticheat. Just copy Hypixel's system, it works.
